frontend/test/pytest/test_gradient.py

```diff
@@ -786,6 +786,28 @@ def workflow(p: float):
     assert np.allclose(result, reference)
 
 
+@pytest.mark.xfail(reason="should pass once #1568 is merged")
+@pytest.mark.parametrize("gate_n_inputs", [(qml.CRX, [1]), (qml.CRot, [1, 2, 3])])
+def test_ps_four_term_rule(backend, gate_n_inputs):
+    """Operations with the 4-term shift rule need to be decomposed to be differentiated."""
```

Comment:
@erick-xanadu I think this test should pass with your PR, it's currently failing.